contributions.md

```diff
@@ -0,0 +1,14 @@
+# Contributions
+
+## Contributors
+We appreciate all of our contributors!
+Each contributor below has made a statement of how they feel they've contributed to `tedana`.
+- Logan Dowdle helps folks get multi-echo data collection going on their scanners, tries to keep up with the increasing number of multi-echo papers, likes making figures that explain what tedana has done to the data, and occasionally adds a new feature (with lots of help!).
```

Comment:
I think it would be good to either include the person's GitHub username after their full name (e.g., `Logan Dowdle (@dowdlelt)`) or make the name a hyperlink (when desired) to a personal site.

Now that GitHub has personal READMEs, I tend to use my profile for a fairly up-to-date personal description.

Reply:
I'm happy to link to people's websites if desired, I just need a link if so.